scripts/obtain/uninstall/dotnet-uninstall-pkgs.sh

```diff
@@ -22,17 +22,17 @@ remove_dotnet_pkgs(){
     
     for i in "${installed_pkgs[@]}"
     do
-        echo "Removing dotnet component - \"$i\""
+        echo "Removing dotnet component - \"$i\"" >&2
```

Comment:
I don't agree that info about expected progress should go to stderr. 

There are plenty of UNIX tools that do this on stdout:

```
$ git fetch --all 2>/dev/null
Fetching origin
Fetching upstream
```